examples/atomic-set/src/main.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import asyncio
+import json
+import logging
+import signal
+
+from sdv_model import Vehicle, vehicle
+
+from sdv.vdb.subscriptions import DataPointReply
+from sdv.vehicle_app import VehicleApp, subscribe_topic
+
+logger = logging.getLogger(__name__)
+logging.getLogger().setLevel(logging.DEBUG)
+
+"""Vehicle Data Broker SetDatapoint examples."""
+
+TOPIC_SET_VALUE_RESPONSE = "vehicleapp/setValue/response"
+TOPIC_SET_VALUE_REQUEST = "vehicleapp/setValue/request"
+# Payload:
+# {"position": 300} -> this request for set seat position
+
+
+class SetMultipleDatapointsAtomiclyApp(VehicleApp):
+    """Speed Limit Warner Vehicle App"""
```

Comment:
Copy & paste issue? App is unrelated to speed limits and warnings.

Reply:
Joh - was also already wrong where I got it from ;-)